docs/database-structure.md

```diff
@@ -0,0 +1,336 @@
+# Schema Public
+
+## Tables
+
+### audit
+
+Some exercises require reviews from other students or any other member of the school. The reviewer checks that some conditions are met by the code of the student being reviewed. This process is called an `audit` and it's kept in this table.
+
+Columns:
+- id.
+- groupId: The group that the audit refers to.
+- auditorId: The user who's making the audit.
+- attrs: The attributes of the audits keep the feedback of the auditor for each question failed.
+- grade: 0 for failed and 1 for passed.
+- createdAt: The timestamp of the time and date of creation.
+- updatedAt: The timestamp of the last update.
+- code: The code that must be used by the auditor to start the audit.
+- resultId: The result of the audit.
+- version: A sha of the commit that the audit refers to.
+- endAt: The last date that the id refers to.
```

Comment:
not sure this is clear - it is used for expiration (after this date the audit is expired). It can also be null, which means that there is no delay to fill the audit

Reply:
Yeah, it's not clear 😅 I guess I got distracted. Thank you